src/components/TodoItem.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+class TodoItem extends React.Component {
+  render() { 
+    return <li>{this.props.todo.title}</li>;
+  }
```

Comment:
- [ ] Please, don't forget to show the `input` which will help you to show every checkbox, as well as the `delete` button to remove an element from the list, as they describe on [Handling event](https://ibaslogic.com/react-form-handling/). :+1: